app/src/main/java/com/example/runningavater/initialFlow/Navigation.kt

```diff
@@ -23,13 +24,13 @@ fun NavGraphBuilder.initialFlow() {
         InitialFlow6Screen()
     }
     composable(route = "initialFlow/7") {
-        InitialFlow7Screen()
+        InitialFlow7Screen(navController = navController)
     }
-    composable(route = "initialFlow/8") {
+    composable(route = "initialFlow/8/{userName}") {
```

Comment:
[must]
app/src/main/java/com/example/runningavater/initialFlow/InitialFlow7.kt の147 行目の修正と合わせて、userNameを渡さないようにする

```suggestion
    composable(route = "initialFlow/8") {
```